c_stubs/xenctrlext_stubs.c

```diff
@@ -435,6 +435,42 @@ CAMLprim value stub_xenctrlext_get_msr_arch_caps(value xch) {
 	CAMLreturn(caml_copy_int64(val));
 }
 
+CAMLprim value stub_xc_cpu_policy_get_system(value xch, value idx, value arr)
+{
+    CAMLparam3(xch, idx, arr);
+    int retval = xc_cpu_policy_get_system(_H(xch), Int_val(idx), &arr);
+    if (retval)
+        failwith_xc(_H(xch));
+    CAMLreturn(Val_int(arr));
+}
+
+CAMLprim value stub_cpu_policy_calc_compatible(value xch, value left, value right)
+{
+    CAMLparam3(xch, left, right);
+    int retval = xc_cpu_policy_calc_compatible(_H(xch), &left &right);
```

Comment:
This won't build you are missing a comma. Again as above you can't pass an ocaml array to C: you'll corrupt memory.